app/views/articles/_v2_form.html.erb

```diff
@@ -1,60 +1,93 @@
+              </button>
+            </div>
+            <div class="crayons-article-form__title">
+              <textarea class="crayons-textfield crayons-textfield--ghost fs-4xl l:fs-5xl fw-bold s:fw-heavy lh-tight" type="text" id="article-form-title" placeholder="New post title here..." autocomplete="off" style="height: 60px; "><%= article.title %></textarea>
+            </div>
+            <div class="crayons-article-form__tagsfield">
+              <input type="text" class="crayons-textfield crayons-textfield--ghost fs-s s:fs-base ff-accent" placeholder="Add up to 4 tags..." name="tags" value="<%= article.cached_tag_list %>" />
+            </div>
+          </div>
         <% end %>
+        <div class="crayons-article-form__body">
+          <div class="crayons-article-form__toolbar">
+            <button class="crayons-btn crayons-btn--ghost crayons-btn--icon-left fw-normal" type="button">
+              <%= (inline_svg_tag("image.svg", aria: true, class: "crayons-icon", title: "Image") + "Upload image").delete!("\n").html_safe %>
+              <input type="file" id="image-upload-field" class="w-100 h-100 absolute left-0 right-0 top-0 bottom-0 overflow-hidden opacity-0 cursor-pointer" multiple accept="image/*" data-max-file-size-mb="25" />
+            </button>
+          </div>
+          <textarea class="crayons-article-form__body__field crayons-textfield crayons-textfield--ghost fs-l ff-accent" id="article_body_markdown" placeholder="Write your post content here..." name="body_markdown"><%= article.body_markdown %></textarea>
+          </div>
       </div>
-      <div class="articleform__buttons">
-        <span style="visibility:hidden;">
-          <p>New Changes <button class="clear-button">clear</button></p>
-          <button class="articleform__buttons--publish">SAVE CHANGES</button>
-        </span>
+      <div class="crayons-article-form__main_aside"></div>
+    </div>
+
+    <div class="crayons-article-form__footer">
+      <button class="crayons-btn mr-2" type="button">Publish</button>
+      <button class="crayons-btn crayons-btn--secondary mr-2" type="button">Save draft</button>
+      <div class="relative">
+        <button class="crayons-btn crayons-btn--ghost crayons-btn--icon" type="button">
+          <%= inline_svg_tag("cog.svg", aria: true, class: "crayons-icon", title: "Options") %>
+        </button>
       </div>
-    </form>
-    <div style="display:none">
-      <% if version == "v2" %>
-        <%= render "pages/editor_guide_text", version: "2" %>
-      <% else %>
-        <%= render "pages/editor_guide_text", version: "1" %>
-      <% end %>
     </div>
-  </div>
+  </form>
+</div>
+
+<div class="hidden">
+  <%= render "pages/editor_markdown_help" %>
+  <%= render "pages/editor_liquid_help" %>
+  <%= render "pages/editor_frontmatter_help" %>
 </div>
 
 <script async>
   <%== RunkitTag.script %>
 </script>
 
 <style>
```

Comment:
This whole `style` area can be moved to the very top of the file since this is style that should be in place _as_ the body renders, not necessarily after (like when before it was just the footer).

So it can go before `<div id="article-form"` at the top.

Reply:
good suggestion ✅